src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1143,6 +1106,17 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			item.texture = "test2";
 			item.renderType = 0;
 			item.customModelName = "Normal";
+			item.isFood = _true;
+			item.nutritionalValue = 5;
+			item.saturation = 0.8f;
+			item.forDogs = _true;

```

Comment:
As we are moving elements, you should name those fields for mojmap mappings, so isMeat, not forDogs, it is a good time to do this now